src/databricks/labs/ucx/hive_metastore/workflows.py

```diff
@@ -36,7 +36,12 @@ def migrate_views(self, ctx: RuntimeContext):
         """
         ctx.tables_migrator.migrate_tables(what=What.VIEW, acl_strategy=[AclMigrationWhat.LEGACY_TACL])
 
-    @job_task(dashboard="migration_main", depends_on=[migrate_views])
+    @job_task(job_cluster="table_migration", depends_on=[migrate_views])
+    def refresh_migration_status(self, ctx: RuntimeContext):
```

Comment:
Adding the refresh inside the migrate tables causes some tests to fail. I concluded that it also makes sense to refresh the migration status after all migration tasks in the job.